blenderproc/python/types/LinkUtility.py

```diff
@@ -443,7 +449,7 @@ def parent_with_bone(self, weight_distribution='rigid'):
                 obj.blender_obj.vertex_groups[0].add(vertices, 1.0, 'REPLACE')
         bpy.ops.object.select_all(action='DESELECT')
 
-    def _create_ik_bone_controller(self, relative_location: Optional[Union[List[float], Vector]] = None,
+    def create_ik_bone_controller(self, relative_location: Optional[Union[List[float], Vector]] = None,
                                    use_rotation: bool = True,
                                    chain_length: int = 0) -> (bpy.types.PoseBone, bpy.types.PoseBone, Matrix):
```

Comment:
return type should be `Optional[Tuple[...]`, `Tuple` is not imported yet.

Reply:
why the `Optional[...]`, and not just `Tuple[...]`?

Reply:
Ah sorry, the collapsed view hides the relevant code :D just Tuple is fine.